poptorch_experimental_addons/cpp/distance_matrix.cpp

```diff
@@ -0,0 +1,498 @@
+// Copyright (c) 2022 Graphcore Ltd. All rights reserved.
```

Comment:
Just to check—has this file had `clang-format` run over it? Asking manually because we haven't added the check to CI yet.

Reply:
Coming soon - #9

Reply:
I had run it, yes, but thanks for including it in the CI!